src/psyclone/tests/domain/lfric/lfric_integer_builtins_test.py

```diff
@@ -142,10 +145,11 @@ def test_int_X_plus_Y(tmpdir, monkeypatch, annexed, dist_mem):
 
 
 def test_int_inc_X_plus_Y(tmpdir, monkeypatch, annexed, dist_mem):
-    ''' Test that 1) the str method of LFRicIntIncXPlusYKern returns the
-    expected string and 2) we generate correct code for the built-in
-    X = X + Y where X and Y are integer-valued fields. Test with and without
-    annexed dofs being computed as this affects the generated code.
+    '''Test that 1) the '__str__' method of 'LFRicIntIncXPlusYKern'
+    returns the expected string and 2) we generate correct code for
+    the built-in 'X = X + Y' where 'X' and 'Y' are integer-valued
+    fields. Test with and without annexed dofs being computed as this
```

Comment:
I would suggest "DoFs" throughout the docstrings.

Reply:
Oops. I thought I'd done that but obviously not. I also found some remaining errors with not using DoFs in `lfric_builtins_test.py` so have fixed those too.

Reply:
Thanks :)